.github/workflows/on_prerelease.yaml

```diff
@@ -0,0 +1,62 @@
+name: Create prerelease
```

Comment:
do we need to separate the prerelease workflow?

Reply:
I followed the task description https://issues.newrelic.com/browse/NR-101028. 

I don't know if it was designed this way to keep the same structure as in `nri-kafka` or if it's because was copy-pasted from the past Kafka task.

Reply:
Updated back to just the release workflow.